### ISSUE TITLE
Faucet cooldown should not be applied if the claim all transaction fails.

### ISSUE THREAD

**Issue by 0xTAY** [curvance-web/app-community#54]
If the 'Claim All' transaction fails users need to wait a full 24 hours before they can try to claim again. We've been getting alot of complaints about this across all networks. 

![Screenshot 2024-09-04 at 12 39 46 PM](https://github.com/user-attachments/assets/fbd38ece-b810-4df0-a57c-187d07262670)

**Comment by joaom00** [comment]
@0xTAY Even after reloading the page, is it still disabled?

**Comment by 0xTAY** [comment]
@joaom00 yup! I tried clearing my cookies too and I still get the cooldown

**Comment by joaom00** [comment]
We get the information to disable it from the blockchain. Either the transaction counted as done or it is expected behavior. @Austinhs, any idea what happened here?
